modules/assignment-objects/src/main/java/com/intuit/wasabi/assignmentobjects/Assignment.java

```diff
@@ -284,6 +284,8 @@ public String toString() {
         EXPERIMENT_IN_DRAFT_STATE(false),
         EXPERIMENT_PAUSED(false),
         NO_PROFILE_MATCH(false),
+        // EXPERIMENT_EXPIRED is no longer used since the experiment is moved to PAUSED upon expiring.
+        // The status is kept for the historical data.
```

Comment:
👍 for adding this comment @AndreaSuckro . Can you also mention that this historical data is only in downstream systems i.e. Hive/Vertica and not in any of the application DBs - Cassandra & MySQL for more clarity?